app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -607,6 +750,54 @@ $('body').on('click', '.reboot-update-btn', function(e) {
   });
 });
 
+// migrate admin node handler
+$('body').on('click', '.trigger-migrate-admin-btn', function(e) {
+  var $btn = $(this);
+  var $modal = $('.migrate-admin-modal');
+
+  // should any node have an out of sync mirror, open the sync modal
+  if(showMirrorSyncModal() > 0) {
```

Comment:
This is preventing `spec/features/admin_node_migration_feature_spec.rb:44` to pass (with the changes we discussed on IRC) as you can see in the screenshot below. I suggest you to also add an expectation to `Migrating` text to indicate that the migration actually started and didn't get stuck in this state.

![error_state-1537983578](https://user-images.githubusercontent.com/188554/46098340-8ed13600-c19a-11e8-9235-0c6ccc77ae4f.png)

Reply:
This is also another good case to have its own test.

Another good scenario would be the disappearance of this RMT/SMT out of sync alert when user clicks on `Confirm`. Locally during my tests it didn't disappear but I didn't investigate why.

Reply:
how did you get to the screenshot ? :D

Reply:
Copied the `after` hook from `automation`.

```ruby
config.after do |example|
  if example.exception
    save_screenshot("screenshots/error_state-#{Time.now.to_i}.png", full: true)
  end
end
```

The only trick was that it had to be declared after the Capybara declarations, so I had to add this to the `capybara.rb` file.